Sources/PostgresKit/PostgresRow+SQL.swift

```diff
@@ -1,8 +1,21 @@
-extension PostgresRow: SQLRow {
-    public func decode<D>(column: String, as type: D.Type) throws -> D where D : Decodable {
-        guard let data = self.column(column) else {
+extension PostgresRow {
+    public func sqlRow(using decoder: PostgresDataDecoder) -> SQLRow {
```

Comment:
Can we keep these internal? We're probably fine just having the `PostgresDataDecoder` one, too

Reply:
This actually doesn't work because the PostgresFluentDriver needs access to this method.

Reply:
Hmm I see. That makes sense. 
Tangent, should we call this method just `sql(decoder:)`? Having row seems redundant since the variable is usually called row: `row.sqlRow`